### PR TITLE
Fhir releases 1.5 New FG's

### PR DESCRIPTION
**Pull request by amadobe**
[PLAT-200604](https://jira.corp.adobe.com/browse/PLAT-200604)
[PLAT-200607](https://jira.corp.adobe.com/browse/PLAT-200607)
[PLAT-203362](https://jira.corp.adobe.com/browse/PLAT-203362)

### CODE REVIEW

File: extensions/industry/healthcare/fhir/fieldgroups/medicationdispense.schema.json [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+   "meta:extensible":true,
+   "meta:abstract":true,
+   "description":"Indicates that a medication product is to be or has been dispensed for a named person/patient.(Based on HL7 FHIR Release 5 specifications)",
+   "definitions":{
+      "medicationDispense":{
+         "properties":{
+            "xdm:healthcareMedicationDispense":{
+               "type":"object",
+               "properties":{
+                  "fhir:identifier":{
```
Comment (anandphatak) [inline extensions/industry/healthcare/fhir/fieldgroups/medicationdispense.schema.json:31]: Pl. change field name from "fhir:identifier" to "xdm:identifier"

File: extensions/industry/healthcare/fhir/fieldgroups/medicationrequest.schema.json [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+   "meta:extensible":true,
+   "meta:abstract":true,
+   "description":"An order or request for both supply of the medication and the instructions for administration of the medication to a patient.(Based on HL7 FHIR Release 5 specifications)",
+   "definitions":{
+      "medicationRequest":{
+         "properties":{
+            "xdm:healthcareMedicationRequest":{
+               "type":"object",
+               "properties":{
+                  "fhir:identifier":{
```
Comment (anandphatak) [inline extensions/industry/healthcare/fhir/fieldgroups/medicationrequest.schema.json:31]: Please change the title from `fhir:identifier` to `xdm:identifier`

File: extensions/industry/healthcare/fhir/fieldgroups/medicationdispense.schema.json (hunk 1 above)
Comment (amadobe) [reply extensions/industry/healthcare/fhir/fieldgroups/medicationdispense.schema.json]: done!

File: extensions/industry/healthcare/fhir/fieldgroups/medicationrequest.schema.json (hunk 2 above)
Comment (amadobe) [reply extensions/industry/healthcare/fhir/fieldgroups/medicationrequest.schema.json]: done!

**Pull request merged**